apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -682,10 +717,9 @@ class TopInstructions extends Component {
             {!this.props.isEmbedView &&
               (isCSDorCSP || this.props.hasContainedLevels) && (
                 <CollapserIcon
-                  collapsed={this.props.collapsed}
+                  isCollapsed={this.props.collapsed}
```

Comment:
NIT: while you're here, would be nice to update to `this.props.isCollapsed` (wow that word is starting to look really weird to me)